popup.js

```diff
@@ -0,0 +1,180 @@
+  },
+  {
+    featuredImg: './assets/Snapshoot Portfolio (2).png',
+    name: 'Uber Navigation',
+    Description:
+        'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    language: ['HTML', 'Ruby', 'CSS', 'JavaScript'],
+    details: { company: 'UBER', profile: 'Front End Dev', year: 2017 },
+    seeLive: 'https://github.com/the-faizmohammad/my-portfolio',
+    seeSource: 'https://github.com/the-faizmohammad/my-portfolio.git',
+  },
+];
+
+function showProjectDetails(index) {
+  const popup = document.getElementById(`card-detail-${index + 1}`);
+  const body = document.getElementById('body');
+  popup.classList.add('card-detail--show');
+  body.classList.add('blur');
+
+  projects.forEach((project, i) => {
+    if (i === index) {
+      project.card.classList.add('active');
+      project.popup.classList.add('active');
+    } else {
+      project.card.classList.remove('active');
+      project.popup.classList.remove('active');
+    }
+  });
+}
+
+function hideProjectDetails(index) {
+  const popup = document.getElementById(`card-detail-${index + 1}`);
+  const body = document.getElementById('body');
+  popup.classList.remove('card-detail--show');
+  body.classList.remove('blur');
+
+  projects.forEach((project, i) => {
+    if (i === index) {
+      project.card.classList.remove('active');
+      project.popup.classList.remove('active');
+    }
+  });
+}
+
+function renderProjects() {
+  const container = document.getElementById('works-cards');
+  const popupContainer = document.getElementById('popup-container');
+
+  projects.forEach((project, index) => {
+    const mycard = document.createElement('li');
+    mycard.classList.add('my-works-card');
+    mycard.id = `my-works-card-${index + 1}`;
+    mycard.innerHTML = `<img src='${project.featuredImg}' alt='Recent Work'>
+        <div class='works-card-main'>
+            <h3 class='works-card-heading-large'>${project.name}</h3>
+            <div class='d-flex works-card-client'>
+                <p class='paragraph'>${project.details.company}</p>
+                <i class='fa-solid fa-circle my-works-card-client-counter'></i>
+                <p class='my-works-card-client-role paragraph'>${
+  project.details.profile
+}</p>
+                <i class='fa-solid fa-circle my-works-card-client-counter'></i>
+                <p class='my-works-card-client-year paragraph'>${
+  project.details.year
+}</p>
+            </div>
+            <p class='paragraph works-card-paragraph'>${project.Description}</p>
+            <ul class="works-card-skills">
+                <li class="skill-list">${project.language[0]}</li>
+                <li class="skill-list">${project.language[1]}</li>
+                <li class="skill-list">${project.language[2]}</li>
+            </ul>
+            <button onclick="showProjectDetails(${index + 1})">See Project</button>
+        </div>`;
+
+    const popup = document.createElement('div');
```

Comment:
- [Required] Good job fixing the issues raised by the previous reviewer, however, there are some issues that were introduced when making the changes. Currently, 
    - when I click on the `See project` button a popup is opened, this popup has the details of the next project in the list instead of the project that was clicked.
    - When I close the popup I realize that there are two popups on the page, please not that the there should be only one popup on the page and it should have the details of the project that was clicked.

Please see the video below and rectify this.

## Video
https://github.com/the-faizmohammad/my-portfolio/assets/16392046/53cb9ca9-8cc6-4eef-9b51-283746835381